backend/functions/chat/chat.py

```diff
@@ -0,0 +1,94 @@
+"""
+url for local testing:
+http://127.0.0.1:5001/schemessg-v3-dev/asia-southeast1/chat_message
```

Comment:
Every request to chat endpoint takes at least 4 seconds in my local setup. This is quite concerning. How long does it take for yours?

Reply:
"say hello" took ~2-3s, but something longer will take longer... yeah on hindsight there's probably some bottleneck somewhere